rdt_visitor.go

```diff
@@ -145,7 +145,8 @@ func (visitor *RdtVisitor) VisitUnion(ctx *rdt.UnionContext, target *UnknownShap
 }
 
 func (visitor *RdtVisitor) VisitGroup(ctx *rdt.GroupContext, target *UnknownShape) (Shape, error) {
-	return visitor.Visit(ctx.GetChildren()[0].(antlr.ParseTree), target)
+	// First and last nodes are terminal nodes
+	return visitor.Visit(ctx.GetChildren()[1].(antlr.ParseTree), target)
```

Comment:
What if GetChildren would return 0 items? Can we handle it?

Reply:
That's impossible, expression parser will fail before visitor.
![image](https://github.com/user-attachments/assets/8fbf4c90-40ff-4078-a0ec-5ff190e2d16d)